pkgs/by-name/ne/netop/package.nix

```
@@ -0,0 +1,27 @@
+{ lib, libpcap, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "netop";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "ZingerLittleBee";
+    repo = pname;
```

As pname is not considered the same as the repo name I suggest changing it to this:
```suggestion
    repo = "netop";
```